### PR TITLE
Migrate `components/formulas`

### PR DESCRIPTION
**Pull request by Etheryte**
## What does this PR change?

Migrate all components in `components/formulas` to Typescript.  
These are pretty messy files to begin with, with a lot of things thrown into one pile.  
For now, I opted to only make this pass compilation without trying to fix the architectural problems.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Part of https://github.com/SUSE/spacewalk/issues/13740

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by ncounter — APPROVED** [review]
I completely agree with 
> These are pretty messy files to begin with, with a lot of things thrown into one pile.

It is out of this PR scope anyway so, this LGTM :+1:

File: web/html/src/components/formulas/EditGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,403 @@
+/* eslint-disable */
```
Comment (cbbayburt) [inline web/html/src/components/formulas/EditGroup.tsx:1]:
```suggestion
```

File: web/html/src/components/formulas/PasswordInput.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
```
Comment (cbbayburt) [inline web/html/src/components/formulas/PasswordInput.tsx:1]:
```suggestion
```

**Review by cbbayburt — APPROVED** [review]
LGTM :+1: 
However, I'm not familiar with the formulas code at all, so I might have missed something. Maybe one of the original authors should also take a look.

Below I pointed out some leftover `eslint-disable` directives

File: web/html/src/components/formulas/EditGroup.tsx (hunk 1 above)
Comment (Etheryte) [reply web/html/src/components/formulas/EditGroup.tsx:1]: Hmm, looks like the migration script isn't cleaning these up correctly. I'll have to have a look later. I cleaned these up manually for now.

**Pull request merged**